src/utilities.ts

```diff
@@ -24,6 +24,12 @@ export function trimStart(target: string, trim: string): string {
   return target;
 }
 
+export function spacesValidation(text: string): boolean {
+  //  let trimmed = text.trim();
+
+  return !text.split("").some(x => x == " ");
```

Comment:
Ah - I gave you the wrong one-liner 😄 

How about: `return !text.includes(" ");`